stock_product_set/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023-2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Add product sets in pickings",
+    "version": "16.0.1.0.0",
+    "category": "Warehouse",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["stock", "sale_product_set"],
```

Comment:
```suggestion
    "depends": ["stock", "product_set"],
```

Reply:
`sale_product_set` module v15 was splitted in v16 (https://github.com/OCA/product-attribute/pull/1384). 
This module should now depend only on `product_set`. Remove everything related to `sale` (no longer needed).

Reply:
The wizard that opens the "Add set" button in planned transfer is defined in the module sale_product_set so it must still depend on sale_product_set even if the logic is in product_set.

https://github.com/OCA/sale-workflow/blob/16.0/sale_product_set/wizard/product_set_add.py#L9

Reply:
Dependency removed